apps/theme/app/themebuilder/_components/ThemePages.module.css

```diff
@@ -1,4 +1,14 @@
 .panel {
+  background-color: var(--ds-color-neutral-background-subtle);
+  border: 1px solid var(--ds-color-neutral-border-subtle);
+  border-radius: 8px;
+  margin-top: 28px;
+  width: 100%;
+  max-width: 1390px;
+  overflow: hidden;
+}
+
+.basicPanel {
   background-color: var(--ds-color-neutral-background-subtle);
```

Comment:
```suggestion
  background-color: var(--ds-color-neutral-background-tinted);
```
This does not change color, but it probably should. It's the one in the back
![image](https://github.com/user-attachments/assets/8b443157-802a-439b-89d2-0e914f7aeb9c)

Reply:
The same also applies to the preview on the frontpage
![image](https://github.com/user-attachments/assets/c8ac016b-cdb2-49c4-a479-3334d13d63b4)